AsyncPoco.Tests/AsyncPoco.Tests.csproj

```diff
@@ -80,19 +85,20 @@
     <Compile Include="MultiPocoTests.cs" />
     <Compile Include="Misc.cs" />
     <Compile Include="ColumnMapper.cs" />
-    <Compile Include="PetaTest.cs" />
     <Compile Include="Program.cs" />
-    <Compile Include="SqlBuilder.cs" />
+    <Compile Include="SqlBuilderTests.cs" />
```

Comment:
I renamed SqlBuilder.cs to SqlBuilderTests.cs to make a little clearer about the content.